view/statistics-flow-rejections.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
 var _                            = require('mano').i18n.bind('View: Statistics')
-  , env                          = require('mano').env
-  , location                     = require('mano/lib/client/location')
-  , selectService                = require('./components/filter-bar/select-service')
-  , tableColumns                 = require('./components/statistics-rejections-table-columns')
```

Comment:
If we would use the hashes instead of array's of values then we will probably need some simple table columns definition for display. Or a mapper of some sorts.